base/commands/map/map_destroy.go

```diff
@@ -0,0 +1,65 @@
+package _map
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/hazelcast/hazelcast-go-client"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc"
+	. "github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/prompt"
+)
+
+const (
+	mapDestroyFlagAutoYes = "yes"
```

Comment:
Could you move this to `clc/const.go`? We will use it with other dangerous commands too.